<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1000</width>
      <height>900</height>
     </rect>
    </property>
    <widget class="QWidget" name="mainPage">
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>280</y>
        <width>861</width>
        <height>591</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="ClickableLabel" name="goTheory">
         <property name="font">
          <font>
           <family>Nunito</family>
           <pointsize>28</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Теория</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ClickableLabel" name="goDemo">
         <property name="font">
          <font>
           <family>Nunito</family>
           <pointsize>28</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Практикум</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ClickableLabel" name="goExam">
         <property name="font">
          <font>
           <family>Nunito</family>
           <pointsize>28</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Проверка знаний</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="Theory" name="theoryPage"/>
    <widget class="Demonstration" name="demoPage"/>
    <widget class="Exam" name="examPage"/>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Theory</class>
   <extends>QWidget</extends>
   <header location="global">theory.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Demonstration</class>
   <extends>QWidget</extends>
   <header location="global">demonstration.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Exam</class>
   <extends>QWidget</extends>
   <header location="global">exam.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header location="global">clickablelabel.h</header>
   <slots>
    <signal>clicked()</signal>
    <slot>on_theory_clicked()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
